SConstruct

```diff
@@ -1380,6 +1388,12 @@ env.AlwaysBuild(
             env.PrettyCommand('TIDY', 'src', 'yellow')
         )))
 
+conf = Configure(env, custom_tests=env.CustomTests)
+
+conf.FindPkgConfigPath()
+
+env = conf.Finish()
```

Comment:
Can we move conf.FindPkgConfigPath() call to the end of this block above?

https://github.com/roc-project/roc/blob/05b998cce82fcc4e6ba9de96ab68821075747606/SConstruct#L1092-L1108